web/src/components/atoms/FormField/styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+import { RED, TURQUOISE } from 'Constants/colors'
+
+const WrapperField = styled.div`
+  margin-bottom: 15px;
+
+  input {
```

Comment:
why the hell do you use styles components than?????
there should be no nesting here. only for pseudo